foundations/html_css/flexbox/flexbox-growing-and-shrinking.md

```diff
@@ -66,6 +66,21 @@ In practice you will likely not be using complex values for `flex-grow`, `flex-s
 
 It _is_ possible to get fancy, and set up layouts where some columns relate to each other in a specific ratio, so it's useful to know that you can use other values, but those are relatively rare.
 
+### Assignment
+
+<div class="lesson-content__panel" markdown="1">
+
+1. Read <a href ="https://www.w3.org/TR/css-flexbox-1/#flex-common">this section</a> to understand the basic values of common flex shorthand values
+
+2. This <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/flex">MDN Doc</a> summarizes the entire flex short hand values as well as introduces some new syntax that hasn't been covered in the previous 2 articles.
```

Comment:
**question:** What does it mean when the bullet refers to "previous 2 articles"? Should this just be "previous article" or am I missing something? Let me know, thanks!

Reply:
Oh it is meant to be previous article, You're right. I'll add a commit to this